src/acceptor.rs

```diff
@@ -12,8 +12,8 @@ use crate::rslice::{rustls_slice_bytes, rustls_str};
 use crate::server::rustls_server_config;
 use crate::{
     ffi_panic_boundary, free_box, rustls_result, set_boxed_mut_ptr, to_box, to_boxed_mut_ptr,
-    try_arc_from_ptr, try_callback, try_mut_from_ptr, try_ref_from_ptr, try_take, Castable,
-    OwnershipBox,
+    try_arc_from_ptr, try_callback, try_clone_arc, try_mut_from_ptr, try_ref_from_ptr, try_take,
```

Comment:
This stale import of `try_arc_from_ptr` fails now that the macro doesn't exist:

```suggestion
   try_callback, try_clone_arc, try_mut_from_ptr, try_ref_from_ptr, try_take,
```